Docs 5776 wsdl and raml strict validation pa (#139)
mulesoft/docs-api-manager: modules/ROOT/pages/access-client-app-id-task.adoc

include::_attributes.adoc[]
endif::[]

> As a client application owner of a registered client, you might be required to provide client application credentials when accessing an API. After a contract is created between the client application and the API, you can access the API using these credentials based on the policies that are applied. For example, if the API owner has applied the Client ID Enforcement policy on the API, you must provide the client ID, and optionally, the client secret information from the client application. 

Reviewer: I'm confused by "As a client application owner of a registered client." Do we mean "the application owner of a registered client"? "a registered client application owner"? Is this someone who owns an application that belongs to a registered client?

Reviewer: Here is my best effort to shorten and clarify:

```suggestion
If you own an application of a registered client, you might be required to provide credentials when accessing an API. For example, if an API owner has applied the Client ID Enforcement policy on an API that you want to access, you must provide the client ID and, optionally, the client secret information. 
```
However, it is not clear to me how being the owner of a registered client application matters. Is it because one might think that a REGISTERED client would not have to provide authentication? If so, maybe that needs stating, something like "Even if you are a registered client, you must provide credentials to access an API"?

Reviewer: Is "secret information" the same as a password? Would it be just as accurate to simplify the requirement to "...your client ID and, optionally, password"?

Author: Kim, it is Client ID and Client Secret. These are two fields in the UI and also referred to as data as such. I saw your confusion and went with init capping these. Client ID and and Client Secret.
